app/assets/javascripts/components/conference/conference_page_overlay.es6.jsx

```diff
@@ -23,13 +23,11 @@ class ConferencePageOverlay extends PageOverlay {
   // Render
   // --------------------------------------------------
   renderModal() {
+    var pairing = this.props.pairing;
     var template = this.props.template;
-    if (this.props.pairing) {
-      return <ConferenceEditModal pairing={this.props.pairing} />;
-    } else if (this.props.template.model == 'group') {
-      if (!template.attributes['leaderships_attributes']) {
-        template.attributes['leaderships_attributes'] = [null, null];
```

Comment:
It's set on template instantiation here: https://github.com/calblueprint/edge-yl/pull/1016/files#diff-196ac8815f97e973c0b9b9e9cc0d8abbR23